src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_SIZE = 100;
+    private Random randomFigure = new Random();
+    private Random randomParameter = new Random();

```

Comment:
```suggestion
    private Random random = new Random();
```

Reply:
not fixed. It's enough to use one random object.